content/docs/pipelines/pipelines-tutorial.md

```diff
@@ -0,0 +1,616 @@
+    * If you own/manage the domain or a subdomain with [Cloud DNS][dns]
+      then you can configure this process to be much faster.
+    * While you wait you can access Kubeflow services by using `kubectl proxy`
+      and `kubectl port-forward` to connect to services in the cluster.
+
+### Create a Cloud Storage bucket
+
+The next step is to create a Cloud Storage bucket to hold your trained model
+
+[Cloud Storage][cloud-storage] is a scalable, fully-managed object/blob store.
+You can use it for a range of scenarios including serving website content,
+storing data for archival and disaster recovery, or distributing large data
+objects to users via direct download. This tutorial uses Cloud Storage to
+hold the trained machine learning model and associated data.
+
+Use the [`gsutil mb`][gsutil-mb] command to create a storage bucket. Your
+*bucket name* must be unique across all of Cloud Storage.
+The following commands create a bucket in the `us-central1` region,
+which corresponds to the `us-central1-c` zone used earlier
+in the tutorial:
+
+```
+export BUCKET_NAME=${PROJECT}-${DEPLOYMENT_NAME}-bucket
+gsutil mb -c regional -l us-central1 gs://${BUCKET_NAME}
+```
+
+## Prepare your pipeline
+
+To simplify this tutorial, you can use a set of prepared files that include
+the pipeline definition and supporting files. The project files are in the 
+[Kubeflow examples repository](https://github.com/kubeflow/examples/tree/master/pipelines/mnist-pipelines)
+on GitHub.
+
+### Download the project files
+
+Clone the project files and go to the directory containing the MNIST pipeline
+example:
+
+```
+cd ${HOME}
+git clone https://github.com/kubeflow/examples.git
+cd examples/pipelines/mnist-pipelines
+```
+
+As an alternative to cloning, you can download the 
+[Kubeflow examples repository zip file](https://github.com/kubeflow/examples/archive/master.zip).
+
+### Set up Python
+
+You need **Python 3.5 or above**. If you don't have Python 3 set up, install
+[Miniconda](https://conda.io/miniconda.html) as described below:
+ 
+* In a Debian/Ubuntu/[Cloud shell](https://console.cloud.google.com/cloudshell) 
+  environment, run the following commands:   
+
+    ```bash
+    apt-get update; apt-get install -y wget bzip2
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    bash Miniconda3-latest-Linux-x86_64.sh
+    ```
+
+* In a Windows environment, download the 
+  [installer](https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe) 
+  and  make sure you select the "*Add Miniconda to my PATH environment variable*" 
+  option during the installation.
+
+* In a Mac environment, download the 
+  [installer](https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh) 
+  and run the following command:
+
+    ```bash
+    bash Miniconda3-latest-MacOSX-x86_64.sh
+    ```
+
+Create a clean Python 3 environment:
+ 
+```bash
+conda create --name mlpipeline python=3.7
+source activate mlpipeline
```

Comment:
Does anaconda not support regular python virtual environments? It seems to me like it would be better to show the more standard `python3 -m venv ./mlpipeline` for people that use a standard python environment

Reply:
I don't know the answer to this one. I'm following the pattern that we currently use in the Pipelines guide:
https://www.kubeflow.org/docs/pipelines/build-pipeline/

For this tutorial, I'll keep it as it is, since it worked for me and since it follows the above pattern.

I've raised https://github.com/kubeflow/website/issues/546 to tackle the issue of the conda environment setup across the docs.